lib_test/expect/test_pcre_split.ml

```diff
@@ -4,20 +4,20 @@ let split ~rex s = Re.Pcre.split ~rex s |> strings
 
 let%expect_test "split" =
   split ~rex:re_whitespace "aa bb c d ";
-  [%expect {| ["aa"; "bb"; "c"; "d"; ""] |}];
+  [%expect {| ["d"; "c"; "bb"; "aa"] |}];
```

Comment:
I think this is wrong; the output is reversed here, but not if the delimiter is present at the beginning of the string (eg in other test cases)